install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -182,7 +182,7 @@ deployment due to configuration errors.
 +
 [NOTE]
 ====
-The GlusterFS playbooks are not guaranteed to be idempotent. Running the playbooks more than once for a given installation is currently not supported without deleting the entire GlusterFS installation (including disk data) and starting over. 
+The GlusterFS playbooks are not guaranteed to be idempotent. Running the playbooks more than once for a given installation is currently not supported without deleting the entire GlusterFS installation (including disk data) and starting over.
```

Comment:
I'm not too sure the second sentence above is saying. Does running the playbook again delete the entire cluster? Or do you have to do that to make any changes? Maybe:

"The GlusterFS playbooks are not guaranteed to be idempotent. To change a GlusterFS configuration, you must delete the entire GlusterFS installation (including dick data) and reinstall."

But that's not what idempotent means, so...should this be two separate admonitions?

Reply:
Fixing in the follow-up PR.